apps/base-docs/base-camp/docs/deployment-to-testnet/contract-verification-sbs.md

```diff
@@ -1,75 +1,74 @@
 ---
 title: Contract Verification
-description: Verify your contract and interact with it.
+description: Verify your contract and interact with it on sepolia.basescan.org.
 hide_table_of_contents: false
 ---
 
-Once your contract is deployed, you can verify it using a number of popular services. Doing so will let you users have confidence that your contract does what you claim, and will allow you to interact with it using a similar interface to what you used in Remix.
+After deploying your contract, you can verify it using popular explorers like [basescan](https://sepolia.basescan.org). Verifying your contract allows users to trust that it does what you claim and enables you to interact with it using a user-friendly interface similar to Remix.
 
 ---
 
 ## Objectives
 
-By the end of this lesson you should be able to:
+By the end of this lesson, you will be able to:
 
-- Verify a contract on the Base Sepolia testnet and interact with it in Etherscan
+- Verify a contract on the Base Sepolia testnet
+- Interact with your verified contract on [sepolia.basescan.org](https://sepolia.basescan.org)
 
 ---
 
-### Verify the Contract
+### Verifying Your Contract
 
-Make sure you still have the address of the contract you deployed in the last article copied to the clipboard.
+First, ensure you have the address of the contract you deployed in the previous article copied to your clipboard.
 
-You can interact with your deployed contract using Remix, the same as before, but it's also possible to interact with it through Etherscan. Paste your address in the search field to find it.
+While you can interact with your deployed contract using Remix, you can also do so through [sepolia.basescan.org](https://sepolia.basescan.org). To find your contract, paste its address into the search field and click search. 
```

Comment:
Please change to reference-style links.  See below for an example!

Reply:
@kaseLunt